src/views/puzzles/PuzzlesExplore/PuzzlesExplore.vue

```diff
@@ -108,13 +114,12 @@
   })
   export default class PuzzlesExplore extends Mixins(PageDataMixin(fetchPageData)) {
     get puzzles() {
-      return get(this.pageData, 'puzzles', []);
+      return this.pageData?.puzzles;
     }
 
     puzzleCleared(id: number) {
-      return get(this, 'pageData.cleared', [])
-        .map(puzzle => puzzle.id)
-        .includes(id);
+      const puzzlesCleared = this.pageData.cleared || [];
+      return puzzlesCleared.map((puzzle: PuzzleItem) => puzzle.id).includes(id);
```

Comment:
I'm against putting types in lambdas, but changing this may be out of scope for this PR.

Reply:
Could we have typed puzzlesCleared as `PuzzleItem[]`?

Reply:
changed